qcoptions.py

```diff
@@ -8,7 +8,7 @@
                   help="Minimum value of scan parameter", metavar="scanmin", default=0)
 parser.add_option("--scanmax", type="int", dest="scanmax",
                   help="Maximum value of scan parameter", metavar="scanmax", default=254)
-parser.add_option("--vfatmask", type="int", dest="vfatmask", 
-                  help="VFATs to be masked in scan & analysis applications (e.g. 0xFFFFF masks all VFATs)", metavar="vfatmask", default=0x0)
+parser.add_option("--vfatmask", type="int", dest="vfatmask",
+		              help="VFATs to be masked in scan & analysis applications (e.g. 0xFFFFF masks all VFATs)", metavar="vfatmask", default=0x0)
```

Comment:
are these tab characters (e.g. \t) here?  If so please change your editor such that when pressing "tab" a fixed number of spaces are inserted.  If you use vim this can be accomplished by editing:

`~/.vimrc`

And placing the following lines:

`
" show existing tab with 4 spaces width
set tabstop=4
" " when indenting with '>', use 4 spaces width
set shiftwidth=4
" " On pressing tab, insert 4 spaces
set expandtab
`

Into the file